security/putty/Portfile

```diff
@@ -2,9 +2,10 @@
 
 PortSystem          1.0
 PortGroup           active_variants 1.1
+PortGroup           cmake 1.1
 
 name                putty
-version             0.70
+version             0.77
 categories          security net
 platforms           darwin
```

Comment:
this is the default since MacPorts v2.7.0 so the line is no longer needed

Reply:
Do you mean "platform"?

Reply:
Yes, sorry for not being clear ;)

Reply:
Fixed